Lib/xml/etree/ElementTree.py

```diff
@@ -679,6 +679,7 @@ def iterfind(self, path, namespaces=None):
     def write(self, file_or_filename,
               encoding=None,
               xml_declaration=None,
+              xml_declaration_definition="<?xml version='{version}' encoding='{encoding}'?>",
```

Comment:
This will break existing code. New positional arguments should be added at the end or as keyword-only.

Reply:
Did'nt know that. will move it to the end.